prymer/primer3/primer3.py

```diff
@@ -371,12 +365,18 @@ def design_primers(self, design_input: Primer3Input) -> Primer3Result:  # noqa:
                 f"Error, trying to use a subprocess that has already been "
                 f"terminated, return code {self._subprocess.returncode}"
             )
-
-        design_region: Span = self._create_design_region(
-            target_region=design_input.target,
-            max_amplicon_length=design_input.params.max_amplicon_length,
-            min_primer_length=design_input.params.min_primer_length,
-        )
+        design_region: Span
+        match design_input.task:
+            case PickHybProbeOnly():
+                design_region = design_input.target
```

Comment:
We should likely validate that `target` is at least as long as the min probe length.

Reply:
this check is now added in `primer3.py` (it requires having access to the `ProbeParameters`, which we do not have in `primer3_task.py`)